packages/api-request-builder/src/delete.js

```diff
@@ -0,0 +1,16 @@
+/* @flow */
+/**
+ * Set the `version` number to the internal state of the service instance
+ * in order to generate a delete uri
+ *
+ * @param  {number} version - The version of the item
+ * @throws if `version` is missing or not a number.
+ * @return {Object} The instance of the service, can be chained.
+ */
+
+export default function toDelete (version: number): Object {
```

Comment:
Why not just name this function `version`? At the end it's exactly what it does.

```js
service.byId('123').version(1).build()

// or
service.byId('123').withVersion(1).build()
```